src/core/reducers/errors.js

```diff
@@ -12,14 +12,21 @@ import { gettext } from 'core/utils';
  * http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#responses
  */
 function getMessagesFromError(error) {
-  let messages = [gettext('An unexpected error occurred')];
+  let data = {
```

Comment:
`data` as a variable name is always a bit of a code smell to me. I know it's not a big difference but even `errorMessageData` or `errorData` would help it feel a bit less generic...